content/information-aggregation/mercattrick-stats.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+/* eslint-disable complexity, no-magic-numbers, max-params */
+
+Foxtrick.modules['MercattrickStats'] = {
+	MODULE_CATEGORY: Foxtrick.moduleCategories.INFORMATION_AGGREGATION,
+	PAGES: ['transferSearchResult'],
+	OPTIONS: ['enableTLPage'],
+
+	CSS: Foxtrick.InternalPath + 'resources/css/mercattrick-stats.css',
+	IMAGES: {
+		LOGO_SMALL: Foxtrick.InternalPath + 'resources/img/mercattrick/logo_small.png',
+		LOGO_XSMALL: Foxtrick.InternalPath + 'resources/img/mercattrick/logo_xsmall.png',
+		FILTERS: Foxtrick.InternalPath + 'resources/img/mercattrick/filters_count.png',
+		BOOKMARKS: Foxtrick.InternalPath + 'resources/img/mercattrick/bookmarks_count.png',
+	},
+	title: 'Mercattrick Transfer Stats',
+	skills: [],
+
+	/**
+	 * @param	{document}	doc
+	 */
+	run: function(doc) {
+        if (Foxtrick.isPage(doc, 'transferSearchResult') && Foxtrick.Prefs.isModuleOptionEnabled('MercattrickStats', 'enableTLPage')) {
+            this.runTL(doc);
+        }
+	},
+
+    /**
+     * @param {document} doc
+     */
+    runTL: async function(doc) {
+        const module = this;
+
+        var playerNodes = Foxtrick.Pages.TransferSearchResults.getPlayerList(doc);
+
+        let ids = playerNodes.map((playerNode) => playerNode.id);
+        // get mercattrick stats
+        try {
+            var data = await Foxtrick.api.mercattrick.getPlayersStats(ids);
+        }
+        catch (err) {
+            return;
+        }
+
+        if (data) {
+            Foxtrick.forEach(function (p) {
+                const statObj = data.find(obj => obj.id === p.id);
+                const filters = statObj ? statObj.filters_count : 0;
+                const bookmarks = statObj ? statObj.bookmarks_count : 0;
+                // creating element
+                const container = Foxtrick.createFeaturedElement(doc, module, 'div');
+
+                Foxtrick.addClass(container, 'ft-mercattrick-stats');
+
+                let img = doc.createElement('img');
+                img.width = 21;
+                img.height = 16;
+                img.title = Foxtrick.L10n.getString('MercattrickStats.title');
```

Comment:
`aria-label` missing

`Foxtrick.addImage` is to be used for backwards- and forwards-compatibility with async resource contexts